tests/test_wave_01.py

```diff
@@ -146,7 +145,7 @@ def test_moves_movie_from_watchlist_to_watched():
     # ****** Add assertions here to test that the correct movie was added to "watched" **********
     # *******************************************************************************************
 
-@pytest.mark.skip()
+# @pytest.mark.skip()
```

Comment:
Hi Poppy! You're missing the additional assert to test that the correct movie was added to `watched`. Let's go over this during our 1:1.

Reply:
Hey Audrey extra required assertion added! Thanks for pointing this out!